### ISSUE TITLE
lore index links are broken for nested guide directories (e.g. tutorial)

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11516]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#1257|
|Type|defect|
|Created|2005-10-08 16:10:11Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1257 1257
type__defect defect
reporter__radix radix
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1128787811000000 1128787811000000
changetime__1128794526000000 1128794526000000
version__ 
owner__ 
cc__radix cc__spiv cc__hypatia
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
If you look at 

http://twistedmatrix.com/projects/core/documentation/howto/tutorial/index.html

check out the Index link at the bottom. It's broken. 

I'm not sure if this is a release bug or a lore bug. I'll investigate.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
It was my bug, I was passing -l ../howto all the time, when the number of "../"s
needed to vary based on the currently-generated howto.

fixed in head, the 2.1 docs will work.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
